runner/test/cases/server/check.test.js

```diff
@@ -0,0 +1,40 @@
+const Code = require("@hapi/code");
+const Lab = require("@hapi/lab");
+const sinon = require("sinon");
+const lab = Lab.script();
+exports.lab = lab;
+const { expect } = Code;
+const { describe, test, afterEach } = lab;
+const fs = require("fs");
+const jsonHelper = require("../../../bin/run/check/getJsonFiles");
+const outOfDateHelper = require("../../../bin/run/check/getOutOfDateForms");
+
+describe("check out of date forms", () => {
+  test("getJsonFiles returns files with .json extension only", async () => {
+    const files = await jsonHelper.getJsonFiles();
+    expect(files).to.contain(["report-a-terrorist.json", "test.json"]);
+    expect(files).not.to.contain(["README.md"]);
```

Comment:
I like this -- good check that we only ever correctly pick up the .json files.

Wonder if we'll ever have any other .json files in here tho? I guess this won't last forever anyway when we start persisting stuff elsewhere

Reply:
well, we do have a readme.md in there already ;). I'm using this as a safety net since I'm doing JSON.parse(..) later down the line